.github/renovate.json

```diff
@@ -38,7 +38,7 @@
         "ci/etc/vcpkg-version.txt$"
       ],
       "matchStrings": [
-        "(<currentValue>^[0-9]{4}\\.[0-9]{2}\\.[0-9]{2}$)"
+        "(?<currentValue>^[0-9]{4}\\.[0-9]{2}\\.[0-9]{2}$)"
```

Comment:
I know not of which I speak, but the `^` looks funny there.

Reply:
Because it is part of the capture?  Seems to work either way, but you are right, it is not very readable, let me send another PR.